.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.20-openshift-4.14
+  tag: rhel-9-release-golang-1.20-openshift-4.15
```

Comment:
main branch CI has moved to 4.16...but clearly this builds and works.  Is this an intentional "use the 4.15 builder because it's the closest matching thing to what the upstream builder image is" ?  Or am I reading too much in here? :smile:

Reply:
That was just me being nervous about 4.16 being unreleased. It's not like that image has anything of non-production quality in it -- it's just a released version of the go toolchain, so no reason for me to be jittery about it.